### PR TITLE
Added usage example to README

### PR DESCRIPTION
**Pull request by io53**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,99 @@
 Default implementation of Ruuvi Bluetooth library for Android
 
 # This link is very useful:
```
Comment (andreevdenis) [inline README.md:4]: Let's change it to "Protocols descriptions" or something more appropriate

File: README.md [hunk 2]
```diff
@@ -2,4 +2,99 @@
...
+import android.os.Bundle
+import android.util.Log
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+import com.ruuvi.station.bluetooth.FoundRuuviTag
+import com.ruuvi.station.bluetooth.IRuuviTagScanner
+import com.ruuvi.station.bluetooth.RuuviRangeNotifier
+
+class MainActivity : AppCompatActivity(), IRuuviTagScanner.OnTagFoundListener {
+    private lateinit var ruuviRangeNotifier: IRuuviTagScanner
```
Comment (andreevdenis) [inline README.md:49]: We should use BluetoothInteractor instead. Probably should make RuuviRangeNotifier private

File: README.md [hunk 3]
```diff
@@ -2,4 +2,99 @@
...
+        ...
+    }
+}
+```
+
+Add the library and its interface to your project by adding following as dependencies in app/build.gradle
+```gradle
+dependencies {
+    implementation 'com.github.ruuvi:com.ruuvi.bluetooth:2cedd919b2'
+    implementation 'com.github.ruuvi:com.ruuvi.bluetooth.default:48cf870335'
```
Comment (andreevdenis) [inline README.md:23]: Let's use 31e717bcb7 version. I fixed few bugs there

Comment (ojousima) [reply README.md:23]: Is there a way to refer to latest release? e.g. com.github.ruuvi:com.ruuvi.bluetooth:1.0.0^
and then make releases on GitHub or something else?

**Comment by ojousima** [comment]
Is this the correct repository for usage instructions? It's my understanding that https://github.com/ruuvi/com.ruuvi.bluetooth has the interface which is implemented by this repository, it would make more sense to me to have the instructions at usage level

File: README.md (hunk 3 above)
Comment (io53) [reply README.md:23]: > Is there a way to refer to latest release? e.g. com.github.ruuvi:com.ruuvi.bluetooth:1.0.0^
> and then make releases on GitHub or something else?

This makes most sense I think. Should we release the current master as 1.0.0? @andreevdenis

Comment (andreevdenis) [reply README.md:23]: I think we can wait till 1.2.12 will be released and we we move 31e717b version of lib to master as 1.0.0.

**Comment by andreevdenis** [comment]
@ojousima how can someone use interface of this lib? Only by writing his own implementation of this lib. Not sure why someone would need it. On other hand by using this repository you can quickly write your own client application and use Ruuvi tags the way you desire.

**Comment by ojousima** [comment]
@andreevdenis Up to you and @io53 as you're actually writing apps : )

My thought was that users would write their client against the interface and then import this implementation, and it would make sense to have usage documentation at the interface level. A bit similar as in C where headers and function documentation are at .h files and actual implementation is at .c files. 

If that model doesn't apply here I guess I don't really understand the purpose of interface repository, but that's a separate issue.

**Comment by io53** [comment]
While the interface is what you'd expect it to be, I agree with @andreevdenis and think it fits better in here. 
If someone is looking into how to use this lib I think they are much more likely to end up looking at this repository rather than the interface.

**Comment by andreevdenis** [comment]
Interface is good from architecture perspective for sure.  For example we can make another version of bt lib using altbeacon or other lib and then use it with current client app without any significant changes. But I doubt anyone would want to create their own bt lib to use with our client. Most likely they would want to create new client and use existing bt lib at least for quick start

**Comment by ojousima** [comment]
Ok to merge by me

**Comment by io53** [comment]
> Ok to merge by me

I'll make the changes suggested by Denis in the weekend and merge after.

**Pull request merged**